src/github.com/matrix-org/dendrite/encryptoapi/storage/encrypt_keys_table.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Vector Creations Ltd
+// Copyright 2018 Vector Creations Ltd
```

Comment:
iirc this needs to be a list and not the latest date if it changes

Reply:
He's just written this file, you're just viewing a subset of commit changes.

@fadeAce If you could change the copyright to:

```
Copyright 2018 X
```

Where X is your name or username.

If you don't want your name, you can just remove the whole line, but leave the remaining license text.

Reply:
sure